wfe/probs.go

```diff
@@ -30,11 +30,11 @@ func problemDetailsForBoulderError(err *berrors.BoulderError, msg string) *probs
 		// not include it.
 		return probs.ServerInternal(msg)
 	case berrors.RejectedIdentifier:
-		return probs.RejectedIdentifier(msg)
+		return probs.RejectedIdentifier(fmt.Sprintf("%s :: %s", msg, err))
 	case berrors.UnsupportedIdentifier:
 		return probs.UnsupportedIdentifier(msg)
```

Comment:
I meant to mention that in the PR desc - I believe `UnsupportedIdentifier` can be removed outright. I did a quick check and it looked like it was only being used for unsupported IDN.